ci.sh

```diff
@@ -157,14 +157,23 @@ download_install_sh() {
 }
 
 install_d() {
-  local install_sh="install.sh"
-  download_install_sh "$install_sh"
-  # DUB isn't needed for gdc
-  if [ "${DMD:-dmd}" == "gdc" ] ; then
-      mkdir -p $HOME/dlang/dub
-      # Remove the check of the lastest DUB version
-      # shellcheck disable=2016
-      sed 's/dub="dub-$(fetch $url)"/dub=dub/' -i "$install_sh"
+  if [ "${DMD:-dmd}" == "gdc" ] || [ "${DMD:-dmd}" == "gdmd" ] ; then
+    export DMD=gdmd
+    if [ ! -e ~/dlang/gdc-8/activate ] ; then
+        sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
+        sudo apt-get update
+        sudo apt-get install -y gdc-8
+        # fetch the dmd-like wrapper
+        sudo wget https://raw.githubusercontent.com/D-Programming-GDC/GDMD/master/dmd-script -O /usr/bin/gdmd
+        sudo chmod +x /usr/bin/gdmd
+        # fake install script and create a fake 'activate' script
+        mkdir -p ~/dlang/gdc-8
+        sudo ln -s /usr/bin/gdc-8 /usr/bin/gdc
```

Comment:
And lastly, if you save the gdmd script to `/usr/bin/gdmd-8`, you wouldn't need this symlink.  I guess you'd need to `export DMD=gdmd-8` ?